packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -560,6 +560,120 @@ function AsyncSortTable() {
+      <Cell>November 27, 2022 at 4:56PM</Cell>
+    </Row>
+    <Row>
+      <Cell>62259692_p0_master1200</Cell>
+      <Cell>PNG</Cell>
+      <Cell>120 KB</Cell>
+      <Cell>January 27, 2021 at 1:56AM</Cell>
+    </Row>
+  </TableBody>
+</TableView>
+```
+
+### Resize events
+
+TableView accepts an `onResize` prop which is triggered whenever a column resizer is moved by the user. This can be used in combination with the `width` prop to update a Column's width in a controlled fashion. TableView also accepts
+an `onResizeEnd` prop as well, triggered when the user finishes a column resize operation. Both events receive a Map object containing the widths of every column in the TableView.
+
+The example below uses `onResize` to update each of the TableView's controlled column widths. It also saves the finalized column widths to `localStorage` in `onResizeEnd`, allowing the TableView's state to be preserved between
+page loads and refreshes.
+
+```tsx example
+let items = [
+  {id: '1', file: '2022-Roadmap-Proposal-Revision-012822-Copy(2)', type: 'PDF', size: '214 KB', date: 'November 27, 2022 at 4:56PM'},
+  {id: '2', file: '62259692_p0_master1200', type: 'PNG', size: '120 KB', date: 'January 27, 2021 at 1:56AM'}
+];
+
+let columnsData = [
+  {name: 'File Name', id: 'file', width: 500},
+  {name: 'Type', id: 'type', width: '1fr'},
+  {name: 'Size', id: 'size', width: 80},
+  {name: 'Date', id: 'date', width: 100}
+];
+
+function ResizableTable() {
+  /*- begin highlight -*/
+  let [widths, setWidths] = React.useState(() => {
+    let localStorageWidths = localStorage.getItem('RSPWidths');
+    return localStorageWidths ?
+      new Map(JSON.parse(localStorageWidths)) :
+      new Map(columnsData.filter(col => col.width).map((col) => [col.id, col.width]));
+  });
+
+  let saveFinalWidths = (e) => {
+    let finalWidths = JSON.stringify([...e]);
+    localStorage.setItem('RSPWidths', finalWidths);
+  };
+
+  // Create new columns object when tracked widths change so Virtualizer rerenders the new column widths
+  let columns = React.useMemo(() => columnsData.map(col => ({...col})), [widths]);
+  /*- end highlight -*/
+  return (
+    <Flex direction="column">
+      <TableView
+        /*- begin highlight -*/
+        onResize={setWidths}
+        onResizeEnd={saveFinalWidths}
+        /*- end highlight -*/
+        aria-label="TableView with controlled, resizable columns saved in local storage"
+        width={800}
+        height={200} >
+        <TableHeader columns={columns}>
+          {(column) => {
+            const {name, id} = column;
+            let width = widths.get(id);
+            return <Column allowsResizing key={id} width={width}>{name}</Column>;
+          }}
+        </TableHeader>
+        <TableBody items={items}>
+          {(item) => (
+            <Row key={item.id}>{(key) => <Cell>{item[key]}</Cell>}</Row>
+          )}
+        </TableBody>
+      </TableView>
+    </Flex>
+  );
+}
+
+<ResizableTable />
+```
```

Comment:
These examples are too wide on mobile and mess up scrolling

Reply:
Updated. Also got rid of one of the columns since the table's columns were almost all the same due to the width restraint, I think the three columns should show how the widths work a bit better without immediately making the TableView scrollable right away